src/main.py

```diff
@@ -606,8 +605,9 @@ def cleanup(signal, frame):   # pylint: disable=redefined-outer-name,unused-argu
 	pHandler.setup()
 	platform.setup()
 
+	logger.info("Checking Internet Connection ...")
 	while not internet_on():
```

Comment:
Because we want the main part of AlexaPi to start after there is internet connection. It can take a while after the system start that the network is available. The systemd unit params should help, but this is a clear check.